dev-util/scons/scons-3.0.5-r1.ebuild

```diff
@@ -42,14 +42,15 @@ src_unpack() {
 	# inside src/ subdirectory to make our life easier
 	if use test; then
 		unpack "${P}.gh.tar.gz"
-		rm -r "${P}/src" || die
+		find "${P}/src" -type f -name '*.py' -delete && echo "Done unpack"
 	else
-		mkdir "${P}" || die
+		mkdir -p "${P}"/src || die
 	fi
 
 	cd "${P}" || die
```

Comment:
Why not just `cd ${P}/src` then, and unpack directly in there?

Reply:
The unpacked archive contains directory "scons-3.0.5" that was early (in scons-3.0.5.ebuild) renamed to 'src'.
So if I 'cd' to 'src' and unpack it here then I will get wrong directory structure.

Reply:
`tar` has cool `--strip-components` option.